extensions/eclipse-che-theia-plugin-ext/package.json

```diff
@@ -15,8 +15,8 @@
     "dependencies": {
         "@eclipse-che/plugin": "latest",
         "@eclipse-che/workspace-client": "^0.0.1-1546509769",
-        "@theia/core": "0.3.18",
-        "@theia/plugin-ext": "0.3.18"
+        "@theia/core": "0.4.0-next.b0064648",
```

Comment:
Probably use 0.3.19 here?

Reply:
yes, need to use tagged version there please (0.3.19 should come next week)

Reply:
I'll change it after the release

Reply:
thanks